sgx-sys/src/lib.rs

```diff
@@ -117,6 +117,7 @@ impl<'e> EnclaveTrait<'e> for Enclave<'e> {
 mod test {
     use super::*;
 
+    #[cfg(has_sgx)]
```

Comment:
This code conditionally compiles the test. We don't want that. We want to conditionally skip (ignore) the test. See how `sev` does this.

Why the difference? Ignored tests are still compiled. There is testing value in ensuring the tests still compile. Further, ignored tests are still included in the testing output, showing explicitly that they were skipped. This helps with debugging.

Reply:
Okay, it makes sense now, thanks. I thought we didn't want to compile it.